src/test/java/seedu/address/logic/parser/FindEventCommandParserTest.java

```diff
@@ -0,0 +1,96 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.EVENT_DATE_DESC_A;
+import static seedu.address.logic.commands.CommandTestUtil.EVENT_DATE_DESC_B;
+import static seedu.address.logic.commands.CommandTestUtil.EVENT_FRIEND_NAME_DESC_A;
+import static seedu.address.logic.commands.CommandTestUtil.EVENT_FRIEND_NAME_DESC_B;
+import static seedu.address.logic.commands.CommandTestUtil.EVENT_NAME_DESC_A;
+import static seedu.address.logic.commands.CommandTestUtil.EVENT_NAME_DESC_B;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_EVENT_DATE_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.PREAMBLE_NON_EMPTY;
+import static seedu.address.logic.commands.CommandTestUtil.PREAMBLE_WHITESPACE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DATE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EVENT_NAME;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.logic.parser.ParserUtil.MESSAGE_INVALID_DATE;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.FindEventCommand;
+import seedu.address.model.event.EventFilterPredicate;
+import seedu.address.testutil.EventFilterPredicateBuilder;
+
+public class FindEventCommandParserTest {
```

Comment:
Can add one test case for garbage input, repeated arguments that is not friend in this case (with valid and invalid) to take the last provided, etc.